script/Ciel/Day 10/10_01_CIEL10_4_3.txt

```diff
@@ -503,47 +543,50 @@ seeing it done right in front of me..."
 [sha:8871fadd09e2de21a3dc7350b5aeb20041337d79]{
 -- Page 277, Offset 27820. Glued. Mods: @x.
 -- 今はたいへん反省しています……」
-I feel very remorseful..."
+ I feel terrible about it now..."
 }
 [sha:03b5c1c8775f828095ef42a91237af946b61a538]{
 -- Page 278, Offset 27821.
 -- しゅん、と肩をすくめて反省する先輩。
-Senpai makes herself small from remorse.
+Senpai curls up in shame.
 }
 [sha:a8413745a51869a5a549b970fdd51c8b211342a8]{
 -- Page 278, Offset 27822.
 -- ……口では大げさだと言ったけど、先輩の話がどれほど現実味を帯びた“もしも”なのかは理解できる。
-I know I said that she was exaggerating, but I understand how firmly her talk of a 'what could have happened' was in the realm of possibilities.
+I know I said that she was exaggerating,
+ but I can still appreciate just how plausible that hypothetical of hers was.
 }
 [sha:60e01ca136c2dfbd4fc12d0d7b454e64621782da]{
 -- Page 279, Offset 27823.
 -- たしかに昨夜のアルクェイドはおかしかった。
-For sure, Arcueid behaved strangely last night.
+Arcueid definitely %{i}was%{/i} acting strangely last night.
 }
 [sha:3ab035ae4a71e897c4b90bf5d7878ffa1488c5d3]{
 -- Page 279, Offset 27824.
 -- ……俺もアイツを怒らせてしまったし、しばらく距離をとって、冷静になってくれるのを待つべきかもしれない。
-I made her angry too and I should have maybe stepped back a bit and waited until she calmed down.
+I made her angry too, so maybe I'd better step back and give her some time to cool off a bit as well.
 }
 [sha:94f21376a3883db9d302bc577eb4ad6f3aa8f71d]{
 -- Page 280, Offset 27825.
 -- 「……わかりました。
-"...Very well.
+"Very well.
 }
 [sha:5eb3428d18b6b40efc8f3b3f004ff725225b7088]{
 -- Page 280, Offset 27826.
 -- もうアルクェイドと仲良くしてほしい、とは言いません。先輩とアイツは天敵っぽいですから」
-I won't ask you to get along with Arcueid anymore since you're natural enemies."
+You and Arcueid seem to be natural enemies,
+ so I'll stop asking the two of you to try and get along."
 }
 [sha:9565af3be7d0e345046e1bedaaf7b4905741bb12]{
 -- Page 280, Offset 27827.
 -- 同属嫌悪ですね、と言いたいのをぐっと堪える。
-I wanted to say that birds of a feather hate each other, but held myself back.
+// Nice one Gekimon
+I resist the urge to quip that birds of a feather hate each other.
```

Comment:
Hmmm, isn't this the opposite of how this saying usually goes?

Reply:
That's the point

Reply:
Sometimes trying to force some kind of saying just ends up muddling things up. Bird of a feather don't hate each other, and it doesn't come across as a "pattern break" unless you actually try to focus on the irony.

>I resist the urge to quip that these birds of a feather seemingly %{i}don't%{/i} flock together.

Or something to that effect. Otherwise it just comes off as a nonsense.